pkg/virt-handler/vm.go

```diff
@@ -2560,6 +2565,13 @@ func (d *VirtualMachineController) hotplugSriovInterfaces(vmi *v1.VirtualMachine
 		return nil
```

Comment:
Shouldn't you delete the vm from the `d.sriovHotplugExecutorPool` in this case? All the sriov nics are plugged, the rate limiting backoff should be zeroed.

Reply:
Good catch! 
Thanks for the heads up, done.